src/shared/compounders/AMMCompounderBase.sol

```diff
@@ -352,14 +371,15 @@ abstract contract AMMCompounderBase is ReentrancyGuard, ERC4626 {
     /// @param _platform - The new platform address.
     /// @param _swap - The new swap address.
     /// @param _owner - The address of the new owner.
-    function updateInternalUtils(address _platform, address _swap, address _owner) external {
+    function updateInternalUtils(address _platform, address _swap, address _owner, uint256 _depositCap) external {
```

Comment:
Why don't we have a dedicated method per internal setting? I see the problem that we only want to update one thing but then have to provide 3 other things, which creates 3 unnecessary opportunities to fuck up.

Reply:
It saves space on the contract. Separating it requires a bit of a re-architecture, not sure if it's worth it.